### DISCUSSION TITLE
documentation to install on ubuntu without docker or snap

### DISCUSSION THREAD

**Discussion by ithakaa** [sct/overseerr#1843] [General]
The reason being that users of LXCs, and there are many, are unable to use docker or snap for the deployment of Overseerr without making unacceptable changes which circumvent container security

A document outlining how to install on a standard ubuntu server would be greatly appreciated

**Comment by TheCatLady** [comment]
Docker and snap are the only officially supported install methods at this time.

**Reply by ithakaa** [reply]
Thats a shame, not interested in support.
Thanks

**Reply by TheCatLady** [reply]
You can always clone the repo and build Overseerr yourself if you'd like, but we do not provide support for this so you'd be on your own.

**Reply by ithakaa** [reply]
Without some form of limited documentation doing so would be a challenge

Thanks anyway.
